### PR TITLE
123 Add Discord social media link to header

### LINKED ISSUE #123: Add Discord social media link to header
**Is your feature request related to a problem? Please describe.**
Allow users who are on the app find a community (discord).

**Describe the solution you'd like**
A button in the header with discord logo, use this link for invites "https://discord.gg/3KhdyJkTWT", this link does not expire.

**Describe alternatives you've considered**
In the future social links may change.

**Additional context**
Social Media Link Examples
image

**Technical ideas**
Modular design.

### PR DESCRIPTION
**Pull request by HansonSin**
Closes #123

### Description of changes

- Added Discord link into the header

### Additional context

-

### CODE REVIEW

**Review by keyserj — COMMENTED** [review]
Welcome to the project 🙂 just some nitpicks

Side note: the project uses [conventional commits](https://github.com/amelioro/ameliorate/blob/main/CONTRIBUTING.md#conventional-commits) as a standard for formatting commit messages - in this case, your first one would probably be something like `chore: add discord logos`. But no need to worry about that, it's not a big deal and I haven't set up an automated PR check to enforce it yet

File: web/src/common/components/Layout.tsx [hunk 1]
```diff
@@ -27,6 +27,8 @@ const Layout: NextPage<LayoutProps> = ({ children }) => {
   const theme = useTheme();
   const githubIconSrc =
     theme.palette.mode == "light" ? "/GitHub-Mark-64px.png" : "/GitHub-Mark-Light-64px.png";
+  const discordIconSrc =
+    theme.palette.mode == "light" ? "/icon_clyde_black_RGB.png" : "/icon_clyde_white_RGB.png";
```
[the remark this answers is not in the document]
Comment (HansonSin) [reply web/src/common/components/Layout.tsx]: Renamed. Clyde is the name of the Discord logo 😄

**Review by keyserj — APPROVED** [review]
Nice work
north-star-discord-icon

**Comment by HansonSin** [comment]
> hey @HansonSin thanks for contributing 🙂 we just added an [all-contributors section](https://github.com/amelioro/ameliorate#contributors-) to the README - can you let me know if you prefer _not_ to be recognized in this way?
> 
> @all-contributors please add @HansonSin for code
> 
> 

Thanks for adding me as a contributor! :D

**Pull request merged**